vision/emergent_object/emergent_object.py

```diff
@@ -0,0 +1,61 @@
+import torch
+
+# You may have to install:
+#   pandas
+#   torchvision
+#   tqdm
+#   seaborn
+
+MODEL_PATH = "vision/emergent_object/best.pt"
+
+
+# Function to do detection / classification
+def emergent_object_detection(image, model):
+    # Convert to RGB
+    image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
+
+    # Run the model on the image. Both a file path and a numpy image work, but
+    #   we want to use a numpy image
+    results = model(image)
+
+    # Retrieve the output from the model
+    output = results.pandas().xyxy[0]
+
+    return output
+
+
+# Load the model from the file
+def emergent_object_model():
+    model = torch.hub.load("ultralytics/yolov5", "custom", path=MODEL_PATH)
+    return model
+
+
+if __name__ == "__main__":
+    import cv2
+
```

Comment:
`cv2` import should be at the top of the file since it is used in `emergent_object_detection()`